.travis.yml

```diff
@@ -80,6 +86,8 @@ jobs:
 
 stages:
   - name: test
+  - name: "pre distro"
+    if tag =~ ^v\d
```

Comment:
Not exactly sure but do we maybe need `:` here after `if`, we have it like that in line 92:

`if: tag =~ ^v\d`

Wanna make sure if this is not a typo.

Reply:
Good catch.

Reply:
Indeed.

Reply:
Fixed.

Reply:
⭐